Configure email notification for account locking - In reCAPTCHA for Single Sign-On (#2980)
wso2/docs-is: en/docs/guides/login/configure-recaptcha-for-single-sign-on.md

{! fragments/configure-recaptcha-for-sso.md !}

!!! Info

>     - If the user exceeds the Maximum failed login attempts as well, the  email notification for account locking scenario can be configured by [Configure Email Notifications for Account Locking](../../../guides/tenants/email-account-locking)

Reviewer: ```suggestion
     If the user exceeds the maximum allowed failed login attempts as well, be sure to [configure email notifications for account locking](../../../guides/tenants/email-account-locking).
```
